improver/threshold.py

```diff
@@ -364,9 +366,12 @@ def process(self, input_cube):
                         clip=True,
                     ),
                 )
-                # if requirement is for probabilities below threshold (rather
-                # than above), invert the exceedance probability
-                if "below" in self.comparison_operator["spp_string"]:
+                # if requirement is for probabilities less_than or less_than_or_equal_to
+                # the threshold (rather than greater_than or greater_than_or_equal_to),
+                # invert the exceedance probability
+                if "less_than" in self.comparison_operator["spp_string"]:
+                    truth_value = 1.0 - truth_value
+                elif "less_than_or_equal_to" in self.comparison_operator["spp_string"]:
```

Comment:
This statement has no effect.  If the substring "less_than_or_equal_to" is in the string, then the substring "less_than" is also in the string.  So this `elif` clause can be removed.